pavement.py

```diff
@@ -26,7 +28,7 @@ def run(args):
     else:
         jobs = []
         for i in range(4):
-            p = multiprocessing.Process(target=run_behave_test, args=(args[0], "single", i))
+            p = threading.Thread(target=run_behave_test,args=(args[0], "single",i))
```

Comment:
Also, can you join() the threads as it should be controlled by the main thread?

Reply:
done